test/common/common/hash_test.cc

```diff
@@ -11,6 +11,16 @@ TEST(Hash, xxHash) {
   EXPECT_EQ(17241709254077376921U, HashUtil::xxHash64(""));
 }
 
+TEST(Hash, xxHashWithVector) {
+  EXPECT_EQ(3728699739546630719U, HashUtil::xxHash64(std::vector<std::string>{"foo"}));
```

Comment:
thanks for adding these these tests. Suggest not looking for specific hash values; that makes the test fail if the hasher upstream changes. But you could do something like:
```
EXPECT_NE(hash({"foo"}), hash({"bar"}));
EXPECT_EQ(hash({"foo", "bar"}, hash({"bar", "foo"}));

Reply:
got it.